src/xunit.performance.api/XunitPerformanceHarness.cs

```diff
@@ -92,21 +96,28 @@ public void RunBenchmarks(string assemblyPath)
             if (configuration == null)
                 throw new ArgumentNullException($"{nameof(configuration)} cannot be null.");
 
+            // Make a copy of the ProcessStartInfo to avoid potential bugs due to changes behind the scenes
+            var startInfo = Clone(processStartInfo);
```

Comment:
I don't think you should make this change.  Adding a dependency on JSON for this doesn't feel right, and honestly, if someone wants to make changes to this, that's just abuse of the API.